raft/doc.go

```diff
@@ -30,6 +33,29 @@ using raft.StartNode or start a Node from some initial state using raft.RestartN
   }
   n := raft.StartNode(c, []raft.Peer{{ID: 0x02}, {ID: 0x03}})
 
+To restart a node from previous state:
+
+  storage := raft.NewMemoryStorage()
+
+  // recover the in memory storage from persistent
```

Comment:
Ditto on *in-memory*. That's the only nit I can find to pick, even if I run it through gofmt.